src/formatDate.js

```diff
@@ -8,7 +8,44 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splitSeparator = fromFormat[fromFormat.length - 1];
+  const splitDate = date.split(splitSeparator);
+
+  const joinSeparator = toFormat[toFormat.length - 1];
+
+  // variables of index date fromFormate
+  const fromYYYY = fromFormat.includes('YY')
+    ? fromFormat.indexOf('YY')
+    : fromFormat.indexOf('YYYY');
+  const fromMM = fromFormat.indexOf('MM');
+  const fromDD = fromFormat.indexOf('DD');
+
+  // variables of index date toFormate
+  const toYYYY = toFormat.includes('YY')
+    ? toFormat.indexOf('YY')
+    : toFormat.indexOf('YYYY');
```

Comment:
The logic for determining the index of the year format in `toFormat` is correct, but similar to the previous comment, ensure that `toFormat` always contains one of these formats to avoid potential errors.